### PR TITLE
fix: rdb files can not be saved because of permission issues bug: redis can not save RDB files #318

### LINKED ISSUE #318: bug: redis can not save RDB files
### Provide environment information

_No response_

### Describe the bug

Redis can not persist it's data right now because of a permission issue

### Link to reproduction

_No response_

### To reproduce

Start the docker image build with the state of #316 and wait for one hour. Then redis will try to save it's RDB file and this will fail. It will then try every 6 seconds to do it again.

### Additional information

From my logs

```
2024-04-06 18:07:12 15:M 06 Apr 2024 16:07:12.069 * 1 changes in 3600 seconds. Saving...
2024-04-06 18:07:12 15:M 06 Apr 2024 16:07:12.071 * Background saving started by pid 1362
2024-04-06 18:07:12 1362:C 06 Apr 2024 16:07:12.073 # Failed opening the temp RDB file temp-1362.rdb (in server root dir /app) for saving: Permission denied
2024-04-06 18:07:12 15:M 06 Apr 2024 16:07:12.172 # Background saving error
```

### PR DESCRIPTION
**Pull request by Meierschlumpf**
<br/>
<div align="center">
  
  <h3>Homarr</h3>
</div>

**Thank you for your contribution. Please ensure that your pull request meets the following pull request:**

- [x] Builds without warnings or errors (``pnpm buid``, autofix with ``pnpm format:fix``)
- [x] Pull request targets ``dev`` branch
- [x] Commits follow the [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] No shorthand variable names are used (eg. ``x``, ``y``, ``i`` or any abbrevation)

Closes #318

### CODE REVIEW

File: packages/redis/redis.conf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Directory to store dump.rdb
+dir /app/redis
```
Comment (manuel-rw) [inline packages/redis/redis.conf:2]: Shouldn't we put this in ``/data/redis`` ?
IMO Redis should be persisted

File: packages/redis/redis.conf [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+# Save the data to disk every 60 seconds if at least 1 key changed
+save 60 1
```
Comment (manuel-rw) [inline packages/redis/redis.conf:5]: How did you decide that this is a good interval? Is this a common default value? Why not something lower?

**Review by manuel-rw — CHANGES_REQUESTED** [review]

File: packages/redis/redis.conf (hunk 2 above)
Comment (Meierschlumpf) [reply packages/redis/redis.conf:5]: As discusses we leave it at 60 seconds

File: packages/redis/redis.conf (hunk 1 above)
Comment (Meierschlumpf) [reply packages/redis/redis.conf]: image
Moved it to `/appdata/redis`

**Pull request merged**